packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -563,12 +564,23 @@ class Chromium {
   }
 }
 
-// TODO(andrewkolos): Remove when https://github.com/dart-lang/sdk/issues/56566
-//  is fixed.
 Future<ChromeTab?> getChromeTabGuarded(
```

Comment:
Can you add a dartdoc for how to use this function?

Reply:
Addressed in [refactor: add clarifying docs](https://github.com/flutter/flutter/pull/154366/commits/9fd3e846b7d2cb8761637839cbca769d38e3f0f3)